### PR TITLE
chore(figma-plugin): Integrate the design tokens Figma plugin into the monorepo.

### PR DESCRIPTION
**Pull request by SkaillZ**
### <strong>Pull Request</strong>
Adds a Figma plugin for theme switching and synchronization of Design tokens.

#### Type of PR

Feature

### CODE REVIEW

**Comment by SkaillZ** [comment]
Blocked by #1563 since we don't build the tailwind CSS in Bazel yet and using the provider is a better idea anyway

File: package.json [hunk 1]
```diff
@@ -107,14 +107,15 @@
...
-    "@nrwl/linter": "^10.0.7",
-    "@nrwl/node": "^10.0.7",
-    "@nrwl/nx-plugin": "^10.0.7",
-    "@nrwl/workspace": "^10.0.7",
+    "@nrwl/angular": "^10.0.13",
+    "@nrwl/jest": "^10.0.13",
+    "@nrwl/linter": "^10.0.13",
+    "@nrwl/node": "^10.0.13",
+    "@nrwl/nx-plugin": "^10.0.13",
+    "@nrwl/workspace": "^10.0.13",
```
Comment (SkaillZ) [inline package.json:118]: Bumping this since 10.0.7 has a bug where the `.nxignore` doesn't have any effect

**Comment by SkaillZ** [comment]
The dependencies (npm + fluid-elements) also don't seem to get bundled since I've switched from the default rollup rule from rules_nodejs to our custom one (any ideas @lukasholzer?).

File: apps/figma-design-tokens/BUILD.bazel [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+package(default_visibility = ["//:__subpackages__"])
+
+# Allow any ts_library rules in this workspace to reference the config
+exports_files(
+    [
+        "tsconfig.json",
+        "rollup.config.js",
+        "manifest.json",
+    ],
+    visibility = ["//visibility:public"],
```
Comment (lukasholzer) [inline apps/figma-design-tokens/BUILD.bazel:16]: NIT: Maybe to subpackages

**Review by lukasholzer — DISMISSED** [review]
Awesome! So nicely documented!

File: libs/barista-components/schematics/src/ng-add/__snapshots__/ng-add.spec.ts.snap [hunk 3]
```diff
@@ -468,42 +468,42 @@ Object {
...
+    "@angular/compiler": "~10.1.6",
     "@angular/core": "^8.2.12",
-    "@angular/forms": "~10.0.5",
-    "@angular/platform-browser": "~10.0.5",
-    "@angular/platform-browser-dynamic": "~10.0.5",
-    "@angular/router": "~10.0.5",
+    "@angular/forms": "~10.1.6",
+    "@angular/platform-browser": "~10.1.6",
+    "@angular/platform-browser-dynamic": "~10.1.6",
+    "@angular/router": "~10.1.6",
```
Comment (SkaillZ) [inline libs/barista-components/schematics/src/ng-add/__snapshots__/ng-add.spec.ts.snap:476]: Bumping the NX version causes newer versions to be installed, so I had to update the snapshot to work around tests failing.

**Comment by lukasholzer** [comment]
@SkaillZ please resolve conflicts

**Review by lukasholzer — APPROVED** [review]

**Review by rowa-audil — DISMISSED** [review]
LGTM 🦑

**Review by tomheller — DISMISSED** [review]

**Comment by tomheller** [comment]
Resolved package.json conflicts.

**Review by rowa-audil — APPROVED** [review]

**Pull request merged**